packages/fxa-auth-client/package.json

```diff
@@ -33,6 +33,6 @@
     "fast-text-encoding": "^1.0.4",
     "mocha": "^10.0.0",
     "typescript": "^4.9.3",
-    "webcrypto-liner": "https://github.com/mozilla-fxa/webcrypto-liner.git#6b3ad971b3b1f0d4da3855c6ceee9b3afa9f0eeb"
+    "webcrypto-liner": "https://github.com/dschom/webcrypto-liner.git#aba1a4d2049f4827d1feb2942bac42a252ae8546"
```

Comment:
This needs to be directed back at the mozilla-fxa/webcrypto-liner repo, but I need this change approved. https://github.com/mozilla-fxa/webcrypto-liner/pull/1.